sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_forecasting_v1/query.sql

```diff
@@ -0,0 +1,120 @@
+-- Query for contextual_services_derived.suggest_foreasting_v1
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+-- DAU SHARE BY COUNTRY
+-- Used to scale automated global KPI forecasts down to US clients only
+WITH client_counts AS (
+  SELECT
+    country,
+    submission_date,
+    COUNT(*) AS total_clients,
+    COUNT(
+      CASE
+  -- SUGGEST DESKTOP ELIGIBILITY REQUIREMENTS
+        WHEN normalized_app_name = "Firefox Desktop"
+          AND
+            -- desktop Suggest experiment start -- 12.5% exposure until 2022-09-21
+          (
+            submission_date >= "2022-06-07"
+            AND browser_version_info.major_version > 92
+            AND country IN UNNEST(["US"])
+          )
+          THEN 1
+        ELSE NULL
+      END
+    ) AS eligible_clients
+  FROM
+    telemetry.unified_metrics
+  WHERE
+    mozfun.bits28.active_in_range(days_seen_bits, 0, 1)
+    AND submission_date >= "2022-06-07"
+    AND sample_id < 10
+    AND normalized_app_name = "Firefox Desktop"
+    # desktop KPI = qualified clients
+    AND active_hours_sum > 0
+    AND uri_count > 0
+  GROUP BY
+    country,
+    submission_date
+),
+grand_total AS (
+  SELECT
+    submission_date,
+    SUM(total_clients) AS monthly_total
```

Comment:
This calculates cumulative days of use over the entire month, not the number of unique clients active in a month. Is this what you want?

Reply:
oh, nevermind, I misread. It is DAU. However, the name "monthly_total" is confusing.